src/platform/forms-system/src/js/actions.js

```diff
@@ -242,6 +250,11 @@ export function uploadFile(file, uiOptions, onProgress, onChange, onError) {
       if (req.status >= 200 && req.status < 300) {
         const body = 'response' in req ? req.response : req.responseText;
         const fileData = uiOptions.parseResponse(JSON.parse(body), file);
+        const eventName = uiOptions.gaEventName
+          ? `${trackingPrefix}${uiOptions.gaEventName}`
+          : `${trackingPrefix}file-uploaded`;
```

Comment:
Are we sure that we want to record events for file uploads even if we don't explicitly define the `gaEventName`?

Reply:
I don't think it hurts anything in GA and I think keeping the form events uniform by default is useful.

Reply:
Chatted w/ @nedierecel ...

She said she actually would like to leverage this tracking in other form-apps too.